src/transaction/log_tran_table.c

```diff
@@ -3541,25 +3541,15 @@ logtb_tran_update_unique_stats (THREAD_ENTRY * thread_p, const BTID * btid, int
   if (write_to_log)
     {
       /* log statistics */
-      char undo_rec_buf[3 * OR_INT_SIZE + OR_BTID_ALIGNED_SIZE + MAX_ALIGNMENT];
-      char redo_rec_buf[3 * OR_INT_SIZE + OR_BTID_ALIGNED_SIZE + MAX_ALIGNMENT];
-      RECDES undo_rec, redo_rec;
+      constexpr size_t data_size (OR_BTID_ALIGNED_SIZE + (3 * OR_INT_SIZE));
 
-      undo_rec.area_size = ((3 * OR_INT_SIZE) + OR_BTID_ALIGNED_SIZE);
-      undo_rec.data = PTR_ALIGN (undo_rec_buf, MAX_ALIGNMENT);
+      char buf[data_size + MAX_ALIGNMENT];
+      char *const data = PTR_ALIGN (buf, MAX_ALIGNMENT);
```

Comment:
Since C++11 [alignas](https://en.cppreference.com/w/cpp/language/alignas) is also possible.
```suggestion
      alignas(MAX_ALIGNMENT) char buf[DATA_SIZE];
```

Reply:
Done